pkg/components/velero/restic/restic.go

```diff
@@ -38,7 +38,7 @@ type BackupStorageLocation struct {
 	Provider string `hcl:"provider"`
 	Bucket   string `hcl:"bucket"`
 	Name     string `hcl:"name,optional"`
-	Region   string `hcl:"region,optional"`
```

Comment:
@knrt10 

`Region` cannot be required as its only required for aws provider in the `BackupStorageLocation` but when using (GCP)[https://github.com/vmware-tanzu/velero-plugin-for-gcp/blob/main/backupstoragelocation.md] or (Azure)[https://github.com/vmware-tanzu/velero-plugin-for-microsoft-azure/blob/main/backupstoragelocation.md] it is not required.

The idea of having default seems not right to me.